tests/acceptance/init/default-test.js

```diff
@@ -0,0 +1,125 @@
+import { pathExists, readFile } from 'fs-extra';
+import { expect, bigtest } from '../helpers';
+import { describe, it, afterEach, beforeEach } from 'mocha';
+import cleanupFiles from '../../../lib/init/utils/clean-up-files';
```

Comment:
I didn't notice this before, or I would have mentioned it earlier. There are aliases set up for each folder in `lib`. So you can clean this import up to be `@init/utils/clean-up-files`